js/api/Geofencing.js

```diff
@@ -74,15 +73,37 @@ export function setupGeofences() {
         });
       })
     })
-    connectMapToGeofence()
+    BackgroundGeolocation.on('geofence', crossGeofence);
   })
 }
 
+export function addGeofence(trigger) {
+  const geofence = {
+    identifier: trigger.id,
+    radius: trigger.radius,
+    latitude: trigger.latitude,
+    longitude: trigger.longitude,
+    notifyOnEntry: true,
+    notifyOnExit: true,
+    notifyOnDwell: true,
+    loiteringDelay: 30000
+  };
+  
+  supressNotificationsTimestamp = Date.now() + 5000;
+  BackgroundGeolocation.addGeofence(geofence, function () {
+    console.log("Successfully added geofence.");
+    BackgroundGeolocation.on('geofence', crossGeofence);
```

Comment:
Yes.  https://github.com/transistorsoft/react-native-background-geolocation-android/issues/56 reports a new version 2.0.0 that we may try out.